### ISSUE TITLE
Button Group Accessibility Issue

### ISSUE THREAD

**Issue by mhemesath** [cerner/terra-core#1061]
# Issue Description
Button groups fail accessibility due to invalid aria attribute

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior
Passes accessibility tests

## Current Behavior
```
ARIA attribute is not allowed: aria-selected="false"
```

## Steps to Reproduce

1. Install [Axe Chrome Extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
2. Navigate to a [button group example page](http://engineering.cerner.com/terra-core/#/tests/button-group-tests/medium-button-group)
3. Run axe by going to developer tools and clicking axe tab.

## Environment

* Component Version:
* Browser Name and Version:
* Operating System and version (desktop or mobile):
